apps/docs-app/docs/integrations/schematics/index.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 4
+---
+
+# Angular Schematics
+
+## Overview
+
+Analogjs provides a series of schematics that help automate some of the frequent tasks inside an Analog workspace, like generating a **Page**, a starting point for a **Blog** (work in progress), or generating **Posts** (work in progress). To use these schematics we can use this command:
+
+### Generate a Page
+
+This command will generate a page inside our `pages` folder with minimal configuration out of the box.
+
+```shell
+ng g @analogjs/platform:page --pathname=index --project=/
```

Comment:
```suggestion
ng g @analogjs/platform:page --pathname=index
```

Reply:
i tried this without the project and it generates the file in the wrong place the '/'

Reply:
Hmm, we should resolve that then, or have project default to `/`. In a single project workspace the default project should be used.